performance-tracking/src/main/java/com/rakuten/tech/mobile/perf/runtime/internal/ConfigurationResult.java

```diff
@@ -22,14 +22,24 @@
   @SerializedName("sendHeaders")
   private Map<String, String> header;
 
+  @SerializedName("modules")
+  private Map<String, Boolean> modules;
```

Comment:
making this `Map<String, Boolean>` locks the `modules` schema into `String -> Boolean` mappings for all values. For now that's not a problem, but if we will add other values in the future this implementation will fail to parse, e.g. 
```json
"modules": {
  "enablePerformanceTracking": true,
  "enableRat": true,
  "ratPercentage": 77
}
```
will result in a parsing error.

If instead we use `Map<String, String>` and `modules.containsKey(key) ? Boolean.valueOf(modules.get(key)) : fallback` the implementation stays forward compatible with schema extensions.

Reply:
Ah, that makes sense. I'll update this.